### PR TITLE
fix: use execution_context_class attribute for GraphQLView

### PR DESCRIPTION
**Pull request by Arfey**
Simple change for using execution_context_class as class attribute

```python
class MyGraphQLView(GraphQLView):
    execution_context_class = CustomExecutionContext
```

### CODE REVIEW

File: graphene_django/views.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def __init__(
         self.pretty = self.pretty or pretty
         self.graphiql = self.graphiql or graphiql
         self.batch = self.batch or batch
-        self.execution_context_class = execution_context_class
+        self.execution_context_class = execution_context_class or self.execution_context_class
```
Comment (jaw9c) [inline graphene_django/views.py:115]: We probs want to follow the same pattern as the other attributes (`graphiql`, `batch` etc):

```suggestion
        self.execution_context_class = self.execution_context_class or execution_context_class
```

**Review by jaw9c — CHANGES_REQUESTED** [review]

File: graphene_django/views.py (hunk 1 above)
Comment (Arfey) [reply graphene_django/views.py:115]: I think so this is not a good pattern. class attribute it's something like default value. when we create an instance we need to have approach to redefine it. thats why i use current order.

Comment (jaw9c) [reply graphene_django/views.py:115]: @Arfey Nice, thats a good point - could you update the other attributes to follow that pattern instead?

Comment (Arfey) [reply graphene_django/views.py:118]: done

**Comment by firaskafri** [comment]
@Arfey great PR!
Seems we have some testing errors?

**Review by jaw9c — APPROVED** [review]
Looks like you just need to run the pre-commit hook so that the formatter runs, nice one on the PR

**Pull request merged**